common/src/main/scala/com/gu/recipeasy/models/CuratedRecipeDB.scala

```diff
@@ -0,0 +1,23 @@
+package com.gu.recipeasy.models
+
+//this model is different from CuratedRecipe as it holds a list of Tag ids
```

Comment:
maybe `RawRecipe` and `CuratedRecipe` as you have said in the tests with a line commenting on what each class represents and in what particular state in the process?